schema.graphql

```diff
@@ -153,8 +153,8 @@ type Query {
   """
   schedule(year: Int): Schedule
 
-  """Get Teaching Preference Survey"""
-  survey: TeachingPreferenceSurvey!
+  """Get Teaching Preference Survey for a given User ID"""
+  survey(userid: Int): TeachingPreferenceSurvey!
```

Comment:
I left the same comment on Eduardo's PR https://github.com/SENG499-Company-4/Backend/pull/32#discussion_r902917570. 

It's fine for now since it's easy to test but once the `login` mutation is in main, we can drop the ID parameter